site/src/_pages/docs/3_spec/1_core-specification.mdx

```diff
@@ -1,6 +1,6 @@
-# Core Specification 0.2
+# Core Specification
```

Comment:
```suggestion
# Core
```

Upon reflection I think this suggestion from Alex is cleaner. The whole thing is already in the 'Specification' section of the sidebar/site.